include/swift/Syntax/RawSyntax.h

```diff
@@ -270,13 +277,17 @@ class RawSyntax final
   }
 
   /// Constructor for creating layout nodes
+  /// If NodeId is 0, the next free NodeId is used, if it is passed, the caller
```

Comment:
Isn't `Optional<unsigned>` more readable to describe these two situations?

Reply:
Yeah, that's probably better. I'll change it.